docs/overview/changelog.md

```diff
@@ -3859,7 +3859,7 @@ _Released 2017-02-06_
   This entailed numerous changes to ensure a consistent experience.
 - Added support for (optionally) using PGRoonga to support full-text
   search in all languages (not just English).
-- Added AppFollow, GitLab, Google Calendar, GoSquared, HelloSign,
+- Added AppFollow, GitLab, Google Calendar, GoSquared, Dropbox Sign,
```

Comment:
Generally it's not correct to edit 2017 changelog entries due to a product renaming itself.